app/views/unca_works/edit_fields/_resource_type.html.erb

```diff
@@ -0,0 +1,5 @@
+<%# OVERRIDE: Render as text instead of select to accommodate imported values that aren't included
```

Comment:
Do we need to add the version of hyku we overridding? other than that looks great!

Reply:
I don't think so; we're not overriding behavior in Hyku itself, we're extending it for this one custom work type. I mostly added `OVERRIDE:` in case someone is grepping the project for custom behavior

Reply:
Perfect! Thanks Kiah